### DISCUSSION TITLE
Support for Certificates and Provisioning Profiles

### DISCUSSION THREAD

**Discussion by chris-araman** [mxcl/xcodebuild#45] [Ideas]
It's arguably out of scope, but it seems like it could be useful to expose inputs for code signing certificates and provisioning profiles. These would be specified as repository or organization secrets.

Something along these lines:
https://docs.github.com/en/actions/guides/installing-an-apple-certificate-on-macos-runners-for-xcode-development
https://github.com/ngeri/prepare-signing

**Comment by mxcl** [comment]
Endorsed.

**Reply by chris-araman** [reply]
Cool. I'll see what I can do here.

**Reply by chris-araman** [reply]
https://github.com/mxcl/xcodebuild/pull/49
